### PR TITLE
PHPStan level 9 for ./p/ and lib_rss.php

### PR DESCRIPTION
**Pull request by Alkarex**
And `app/FreshRSS.php`.
Contributes to https://github.com/FreshRSS/FreshRSS/issues/4112 (addresses 146 level-6 errors)
Refactored `greader.php` as a class with private functions.

### CODE REVIEW

File: app/Models/Entry.php [hunk 1]
```diff
@@ -17,10 +17,14 @@ class FreshRSS_Entry extends Minz_Model {
 	 */
 	private $guid;
 
+	/** @var string */
```
Comment (aledeg) [inline app/Models/Entry.php:20]: Can't you add the type directly in the property definition?

Comment (Alkarex) [reply app/Models/Entry.php:20]: That would require PHP 7.4+: https://php.net/language.types.declarations

With PHP 7.2 (which we support at the moment https://github.com/FreshRSS/FreshRSS/pull/4848 ): `PHP Parse error:  syntax error, unexpected 'string' (T_STRING), expecting function (T_FUNCTION) or const (T_CONST) in ./app/Models/Entry.php on line 18`

Comment (Alkarex) [reply app/Models/Entry.php:20]: Let's come back to it in a few months https://github.com/FreshRSS/FreshRSS/discussions/3321#discussioncomment-835704

Comment (Alkarex) [reply app/Models/Entry.php:20]: Interesting reading; there are several details I did not know (e.g. about uninitialized properties) https://wiki.php.net/rfc/typed_properties_v2

**Pull request merged**